### ISSUE TITLE
Make ccm compatible with Manjaro

### ISSUE THREAD

**Issue by severach** [graysky2/clean-chroot-manager#41]
Enclosed is a patch to make ccm compatible with Manjaro. If you want to test it you also need to patch the Manjaro version of arch-nspawn. 

There's more Manjaro functionality that could be added but this is enough to get it to work.

[ccm-manjaro-patches.zip](https://github.com/graysky2/clean-chroot-manager/files/1132717/ccm-manjaro-patches.zip)

````
ccm-manjaro.patch for clean-chroot-manager
manjaro-arch-nspawn.patch for Manjaro arch-nspawn
manjaropatch.sh apply patches to a live Manjaro system
````

**Comment by graysky2** [comment]
Send a PR?

**Comment by f0086** [comment]
Why is this issue closed?
The problem remains and @severach already provided patch files to fix it.
I've opened PRs here and on the devtools repo to get this fixed.

**Comment by graysky2** [comment]
Better to "fix" in devtools since this wrapper script works with them.

**Comment by f0086** [comment]
This can't be fixed in devtools, because manjaro uses a own version of devtools (see here: https://wiki.manjaro.org/index.php?title=Manjaro-tools), so clean-chroot-manager is not compatible with manjaro by design.

I close this PR.

**Comment by severach** [comment]
Read the patch shell script. The patches are for ccm and Manjaro's devtools.

There are no patches for Arch Linux or Antergos. The Arch Linux pull request archlinux/devtools/pull/9 should be closed but #54 should not. The arch-nspawn bug should be filed with [Manjaro](https://gitlab.manjaro.org/tools/development-tools/devtools).
